cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/connectionStrategy/ConnectionStrategy.kt

```diff
@@ -11,7 +11,7 @@ abstract class ConnectionStrategy(protected val vertx: Vertx, protected val even
         return this.getJsonObject(CogboardConstants.Props.ENDPOINT_LOADED)?.getString(prop) ?: ""
     }
 
-    protected open fun authenticationTypes(): Set<AuthenticationType> {
+    protected open fun authenticationTypes(): Set<Any> {
```

Comment:
dlaczego Any?

Reply:
W tym momencie w projekcie są dwa różne enumeratory o nazwie AuthenticationType - jeden od http i ssh, więc poprzednia definicja była błędna bo odnosiła się tylko do jednego z nich. Pewnie jest lepszy sposób na obejście tego problemu, w momencie pisania kodu nie miałem nic lepszego w głowie.

Reply:
Poczytaj jeszcze proszę i przy okazji spróbujmy to naprawić. Na razie puszczam